index.html

```diff
@@ -285,107 +271,211 @@ <h2>
-        the battery is charging, the implementation is unable to report the
-        state, or there is no battery attached to the system, or otherwise.
-        When the battery charging state is updated, the <a>user agent</a> MUST
-        <a>queue a task</a> which sets the <code>charging</code> attribute's
-        value and <a>fires an event</a> named
-        <code><a>chargingchange</a></code> at the <a>BatteryManager</a> object.
-      </p>
-      <p>
-        The <code id="widl-BatteryManager-chargingTime">chargingTime</code>
-        attribute MUST be set to 0, if the battery is full or there is no
-        battery attached to the system, and to the value positive Infinity if
-        the battery is discharging, the implementation is unable to report the
-        remaining charging time, or otherwise. When the battery charging time
-        is updated, the <a>user agent</a> MUST <a>queue a task</a> which sets
-        the <code>chargingTime</code> attribute's value and <a>fires an
-        event</a> named <code><a>chargingtimechange</a></code> at the
-        <a>BatteryManager</a> object.
-      </p>
-      <p>
-        The <code id=
-        "widl-BatteryManager-dischargingTime">dischargingTime</code> attribute
-        MUST be set to the value positive Infinity, if the battery is charging,
-        the implementation is unable to report the remaining discharging time,
-        there is no battery attached to the system, or otherwise. When the
-        battery discharging time is updated, the <a>user agent</a> MUST
-        <a>queue a task</a> which sets the <code>dischargingTime</code>
-        attribute's value and <a>fires an event</a> named
-        <code><a>dischargingtimechange</a></code> at the <a>BatteryManager</a>
-        object.
-      </p>
-      <p>
-        The <code id="widl-BatteryManager-level">level</code> attribute MUST be
-        set to 0 if the system's battery is depleted and the system is about to
-        be suspended, and to 1.0 if the battery is full, the implementation is
-        unable to report the battery's level, or there is no battery attached
-        to the system. When the battery level is updated, the <a>user agent</a>
-        MUST <a>queue a task</a> which sets the <code>level</code> attribute's
-        value and <a>fires an event</a> named <code><a>levelchange</a></code>
-        at the <a>BatteryManager</a> object.
-      </p>
-      <p class="note">
-        The definition of how often the <code><a>chargingtimechange</a></code>,
-        <code><a>dischargingtimechange</a></code>, and
-        <code><a>levelchange</a></code> events are fired is left to the
-        implementation.
-      </p>
-      <section id="multiple-batteries">
-        <h2>
-          Multiple batteries
-        </h2>
+      <aside class="note">
+        If the user agent is <a>unable to report the battery status
+        information</a>, the <a>BatteryManager</a>'s internal slots will remain
```

Comment:
I'd suggest doing this globally: 

```suggestion
        information</a>, the {{BatteryManager}}'s internal slots will remain
```

It just helps distinguish between concepts and WebIDL things.

Reply:
Done.